server/utils/tasks/index.test.ts

```diff
@@ -87,201 +71,116 @@ describe('index', () => {
             },
           },
           {
-            key: {
-              text: 'AP Area',
-            },
-            value: {
-              text: application.apArea.name,
-            },
-          },
-          {
-            key: {
-              text: 'Currently allocated to',
-            },
-            value: {
-              text: task.allocatedToStaffMember.name,
-            },
+            key: { text: 'AP Area' },
+            value: { text: application.apArea.name },
           },
           {
-            key: { text: 'Applicant' },
-            value: { text: getFormattedNameAndEmail(applicantUserDetails.name, applicantUserDetails.email) },
-          },
-          {
-            key: { text: 'Case Manager' },
-            value: { text: getFormattedNameAndEmail(caseManagerUserDetails.name, caseManagerUserDetails.email) },
+            key: { text: 'Currently allocated to' },
+            value: { text: 'Unallocated' },
           },
           {
             key: { text: 'Gender for AP' },
-            value: { text: sentenceCase(application.genderForAp) },
-          },
-          {
-            key: { text: 'Applicant PDU' },
-            value: { text: 'test' },
+            value: { text: 'Male' },
```

Comment:
I'm thinking gender should be man/woman or perhaps for APs, `men's` and `women's`